scripts/terraform-fmt.ts

```diff
@@ -0,0 +1,70 @@
+import { exec, ExecException } from 'child_process';
+import { exit } from 'process';
+
+const BINARY = 'terraform';
+const DIR = 'packages/airnode-deployer/terraform';
+const CMD_CHECK = `${BINARY} fmt -list=true -write=false -recursive ${DIR}`;
+const CMD_WRITE = `${BINARY} fmt -list=true -write=true -recursive ${DIR}`;
+
+const EC_ARGUMENTS = 22; // Invalid arguments
+const EC_TERRAFORM = 23; // Terraform command failed
+const EC_FORMATTING = 24; // Formatting issue
+
+if (process.argv.length !== 3) {
+  console.error('Wrong script usage!');
+  console.error('terraform-fmt.ts check | write');
+  exit(EC_ARGUMENTS);
+}
+
+const command = process.argv[2];
+
+if (!['check', 'write'].includes(command)) {
+  console.error(`Unknown command '${command}'`);
+  exit(EC_ARGUMENTS);
+}
+
+exec(`command -v ${BINARY}`, (err, _stdout, _stderr) => {
```

Comment:
As per your Slack DM/my response, it may be better to stat the file first and/or just directly run it so that this command is compatible with Windows/PowerShell/`cmd.com`.